### ISSUE TITLE
Many, many missing button labels

### ISSUE THREAD

**Issue by tyrylu** [NordicSemiconductor/IOS-nRF-Connect#98]
When you try to use the application with VoiceOver, you notice that there are many, many unlabeled buttons, like the ones near a device in the scan results list, near a characteristic etc. That makes the application almost unusable for visually impaired people. As the sources are not available, if you'd like, i can only provide testing.
